pynguin_0_41_0/src/pynguin/custom_seeding/strategy/test_strategy.py

```diff
@@ -0,0 +1,226 @@
+        for val in values:
+            found = False
+            for result in results:
+                if result[0] == param_name:
+                    result.append(val)
+                    logger.debug("Sorted value %s into the result list %s", val, result)
+                    found = True
+                    break
+            if not found:
+                results.append([param_name, val])
+
+    @staticmethod
+    def find_parameters(node: NodeNG, param_names: list[str],
+                         results: list[list[str]], operations: tuple[str]) -> None:
+        """Extracts string values, specific parameters are checked against.
+
+        A given node is checked, whether it includes a compare operation.
+        If the operation is included in operations,
+        the two values in the operation are written into a left and right part.
+        It is tested, if those are in the given parameters name and given a constant value.
+        These values are then written into results.
+        ATTENTION!!! Constant is double checked here and in _extract_literal_repr
+        meaning that lists do not work currently. This is due to problems caused by other operations
+        i.e. lower that is not currently handled correctly.
+        """
+        if isinstance(node, Compare):
+            for op, comparator in node.ops:
+                # checks whether the node includes operations from the operations parameter
+                if op in operations:
+                    left = node.left
+                    right = comparator
+
+                    # Testcase: param is in something
+                    if (
+                        isinstance(left, Name)
+                        and left.name in param_names
+                        and isinstance(right, Const)
+                       ):
+                        values = TestStrategy._extract_literal_repr(right)
+                        TestStrategy.append_results(left.name, values, results)
+
+                    # Testcase: something is in param
+                    elif (
+                        isinstance(right, Name)
+                        and right.name in param_names
+                        and isinstance(left, Const)
+                        ):
+                        values = TestStrategy._extract_literal_repr(left)
+                        TestStrategy.append_results(right.name, values, results)
+
+    @staticmethod
+    def visit(node: NodeNG, param_names: list[str], tests: list[list[str]], operations: tuple[str],
+              results: list[list[str]] | None = None):
+        """Recursively traverses node tree looking for specific operations, producing testcases.
+
+        Checks a node for specific operations (if with 'or' or 'and').
+        It then traverses the tree in a way that the results are correctly build to
+        guarantee the right way to build the testcase inputs following the extraction.
+        ATTENTION!!! While and for loops create no wrong tests,
```

Comment:
Like mentioned above, there should be no duplicates in the tests handed-back to main_seeder.py